docs/docs/api.md

```diff
@@ -337,6 +348,43 @@ These types are not validated at run time, the result is only cast to the provid
 
 :::
 
+## `    sql`` `
+
+`.sql<T>(strings: TemplateStringsArray, ...params: any[]): Promise<Results<T>>`
```

Comment:
Move this interface line into the sql method section above.

```suggestion
```

This section of the docs is about the "Tagged Template Queries" as a whole, along with the helper function, it could expand if we add more functionality.

The section in the methods section above is highlighting the existence of the sql method, and linking to the expanded set of methods and helpers here.